app/src/main/java/com/nononsenseapps/feeder/archmodel/FeedItemStore.kt

```diff
@@ -143,6 +143,12 @@ class FeedItemStore(
         }
 
         when {
+            filter.search.isNotEmpty() -> {
```

Comment:
I think instead of baking it into the filter, a new parameter should be added to the function